ros/src/tl_detector/tl_detector.py

```diff
@@ -113,6 +115,9 @@ def image_cb(self, msg):
             self.upcoming_red_light_pub.publish(Int32(self.last_wp))
         self.state_count += 1
 
+    def eucl_dist(self, a, b):
+        return math.sqrt((a.x - b.x) ** 2 + (a.y - b.y) ** 2)
```

Comment:
do we need to include z dimension?

Reply:
Hm, I don't think so, z is just the height right? I think the x and y feel good for me as it is?